docs/source/configuration.rst

```diff
@@ -60,6 +63,69 @@ and, therefore, the actions are executed in that same order.
 
 See :ref:`Examples` for configuration file examples.
 
+Defaults
+~~~~~~~~
+
+- The value type of ``defaults`` is a dictionary.
+
+This dictionary must have the following key:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 1 1 2
+
+   * - Key Name
+     - Value Type
+     - Value Description
+   * - ``actions``
+     - dictionary of :ref:`Actions`
+     - A dictionary made of :ref:`Actions` whose configuration will be used by default.
+
+The ``defaults`` section is used to define default configuration valued for actions run by pull request rules and by :ref:`Commands`.
+If the options are defined in ``pull_request_rules`` they are used, otherwise, the values set in ``defaults`` are used.
+
+See example below:
+
+.. code-block:: yaml
+
+  defaults:
+    actions:
+      comment:
+        message: I 💙 Mergify
```

Comment:
Not sure it's the best place to use a `defaults`, I don't expect people to use a default message for comment. :)